src/components/TemplateLibrary/index.js

```diff
@@ -30,6 +33,11 @@ const LibraryComponent = () => {
     load();
   }, []);
 
+  /**
+   * Upload a template locally
```

Comment:
If you're adding a full stop in the **main description** (not the blocks), make sure you do it everywhere.

Reply:
Better description would be `Render the uploaded template.`